broker/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -76,6 +79,12 @@ private void validateConfiguration() {
       throw new IllegalArgumentException(String.format(NODE_ID_ERROR_MSG, nodeId, clusterSize));
     }
 
+    final var maxAppendBatchSize = cluster.getMaxAppendBatchSize();
```

Comment:
Any reason we validate just the batch size and not the max appends?

Reply:
Actually I thought about it as well and just did it because it is the DataSize type where I need to have a smaller value then int max. Whether it is negative will be validated in the Atomix Building